src/components/trades/Trades.vue

```diff
@@ -313,7 +327,21 @@ export default class extends Mixins(PaneMixin) {
         'rgb(' + backgroundColor[0] + ', ' + backgroundColor[1] + ', ' + backgroundColor[2] + ', ' + percentToSignificant + ')'
 
       if (amount > this._audioThreshold) {
-        this._liquidationsAudio[trade.side](audioService._play, amount / (this._significantThresholdAmount * multiplier), trade.side, 0)
+        if (this._liquidationsAudio[trade.side].toString().includes('function anonymous(play,')) {

```

Comment:
appendRow / onTrades are very expensive functions
Thus the need to merge both implementations (AudioFunction and AudioURLFunction)